src/cljc/rems/common/form.cljc

```diff
@@ -183,7 +183,7 @@
 
 (defn- validate-not-present [field key]
   (when (contains? field key)
-    {key :unsupported}))
+    {key {key :t.form.validation/unsupported}}))
```

Comment:
I think you can revert this change now. The frontend shouldn't see any `:unsupported` validations any more.

Reply:
And I guess also the text localizations can be removed?

Reply:
yeah that's ok too, or we can keep the translation key & translations for "defense in depth"

Reply:
I think we should keep it

Reply:
so then just revert to `{key :t.form.validation/unsupported}`